libparsers/src/xml/dynamic_background.cpp

```diff
@@ -41,6 +41,17 @@ namespace xml {
 
 					auto const absolute = get_relative_file(filename, object_filename);
 
+					current_section = current_section->next_sibling("background", 0, false);
+					// If multiple object_backgrounds specified add an error.
+					if (current_section != nullptr) {
+						auto* is_second_object = current_section->first_node("object", 0, false);
+						if (is_second_object != nullptr) {
+							std::ostringstream err;
+							err << "Multiple Object backgrounds: only one object background is "
+							       "allowed.";
+							add_error(errors, filename, 0, err.str());
```

Comment:
You should just make this a string literal (by doing `using namespace std::string literals` and putting an s on the end of the literal, then just pass it directly into the add_error func. Because there is no formatting happening you don't need a ostringstream here.